backend/devices/bitbox/device.go

```diff
@@ -1373,16 +1373,13 @@ func (dbb *Device) ExtendedPublicKey(keypath signing.AbsoluteKeypath) (*hdkeycha
 }
 
 // KeystoreForConfiguration implements device.Interface.
-func (dbb *Device) KeystoreForConfiguration(
-	cosignerIndex int,
-) keystoreInterface.Keystore {
+func (dbb *Device) KeystoreForConfiguration() keystoreInterface.Keystore {
```

Comment:
The method name makes little sense now, tbh.

Reply:
Renamed